heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -452,6 +487,28 @@ public static void setTopologyExactlyOnceEnabled(Map<String, Object> conf, boole
     conf.put(Config.TOPOLOGY_EXACTLYONCE_ENABLED, String.valueOf(exactOnce));
   }
 
+  public static void setTopologyPackingAlgorithm(Map<String, Object> conf,
+                                                 PackingAlgorithmType type) {
```

Comment:
I think the packing impls should be defaulted at the system level, but I certainly see the need to be able to override them at the topology level. We've recommended to some topology owners to specify a different packing impl to change the way resources are allocated for their specific job, which seems reasonable to me.